pkg/cache/sql/Readme.md

```diff
@@ -0,0 +1,167 @@
+# SQL Cache
+
+## Sections
+- [ListOptions Informer](#listoptions-informer)
+  - [List Options](#list-options)
+  - [ListOption Indexer](#listoptions-indexer)
+    - [SQL Store](#sql-store)
+    - [Partitions](#partitions)
+- [How to Use](#how-to-use)
+- [Technical Information](#technical-information)
+  - [SQL Tables](#sql-tables)
+  - [Attach Driver](#attach-driver)
+  - [SQLite Driver](#sqlite-driver)
+  - [Connection Pooling](#connection-pooling)
+  - [Encryption Defaults](#encryption-defaults)
+  - [Indexed Fields](#indexed-fields)
+  - [ListOptions Behavior](#listoptions-behavior)
+  - [Troubleshooting Sqlite](#troubleshooting-sqlite)
+
+
+
+## ListOptions Informer
+The main usable feature from the SQL cache is the ListOptions Informer. The ListOptionsInformer provides listing functionality,
+like any other informer, but with a wider array of options. The options are configured by informer.ListOptions.
+
+### List Options
+ListOptions includes the following:
+* Match filters for indexed fields. Filters are for specifying the value a given field in an object should be in order to
+be included in the list. Filters can be set to equals or not equals. Filters can be set to look for partial matches or
+exact (strict) matches. Filters can be OR'd and AND'd with one another. Filters only work on fields that have been indexed.
+* Primary field and secondary field sorting order. Can choose up to two fields to sort on. Sort order can be ascending
+or descending. Default sorting is to sort on metadata.namespace in ascending first and then sort on metadata.name.
+* Page size to specify how many items to include in a response.
+* Page number to specify offset. For example, a page size of 50 and a page number of 2, will return items starting at
```

Comment:
I assume the page-number is 1-based?

Reply:
If you mean do page numbers start at one, yes. I added some clarification